### ISSUE TITLE
AnimGraph Node Groups rename is difficult to use

### ISSUE THREAD

**Issue by FuzzyCarterAWS** [o3de/o3de#13372]
**Describe the bug**
The anim graph node groups rename field is small and difficult to use.

**Steps to reproduce**
1. Open the Animation Editor
2. Set Animation Editor to **AnimGraph** mode 
3. Select the AnimGraph panel
4. Add a few nodes
5. Right click a node and select **Create Node Group**
6. Attempt to rename the Node Group
   * Right-Click on node group and select rename
   * Double-Click on node group title bar

**Expected behavior**
Node group rename field fits within the node group and shows the text

**Actual behavior**
Node group rename field extends past the node group and is not tall enough to see the text completely.

**Screenshots/Video**
![EFC88A28-D6CB-44DF-AD79-995ACE19BE7D](https://user-images.githubusercontent.com/29718640/203630397-607f58c4-ada9-46d0-a2d5-a704cc5648a4.GIF)

**Found in Branch**
Development

**Comment by Be-ing** [comment]
Hmm, I've never seen this and I haven't figured out how to reproduce it.

**Comment by Be-ing** [comment]
What OS are you testing on?

**Comment by FuzzyCarterAWS** [comment]
@Be-ing turns out that it's a scaling issue.  If I zoom in completely this issue disappears.  To repro you'll have to be zoomed out a bit from the full zoom.  I'll update the description to account for this.

**Comment by hultonha** [comment]
I've assigned @Be-ing to this for now, please free to leave an update if this isn't something you'll be tackling. If it looks hairy I think it's okay to leave for now as it is relatively minor in the grand scheme of things.

**Comment by Be-ing** [comment]
There are two separate issues here:
1. When zoomed out and editing the node group title text, the font size is too big, so the bottom of the text gets cut off.
2. The QLineEdit for editing the text extending outside the group area to the right.

I can reproduce 1 but not 2.

**Comment by hultonha** [comment]
Okay understood, and this is separate to the issue with editing the text when being zoomed in being difficult? That was definitely the more important one to fix

**Comment by Be-ing** [comment]
Yes, I'm looking into the scaling of the text with zooming not working at all. Seems it was trying to scale a point size of -1 :confused:

**Comment by FuzzyCarterAWS** [comment]
@hultonha @Be-ing 

Updated the workflow docs in o3de/sig-simulation#22

**Comment by hultonha** [comment]
> Yes, I'm looking into the scaling of the text with zooming not working at all. Seems it was trying to scale a point size of -1 😕

Interesting 🤔 I hope tracking it down isn't too painful, good luck and godspeed! 😅
